geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/Conversions.scala

```diff
@@ -191,6 +193,11 @@ object RichSimpleFeatureType {
   val DEFAULT_DATE_KEY    = "geomesa.index.dtg"
   val ST_INDEX_SCHEMA_KEY = "geomesa.index.st.schema"
   val USER_DATA_PREFIX    = "geomesa.user-data.prefix"
+  val KEYWORDS_KEY        = "geomesa.keywords"
+
+  val KEYWORDS_SPLITTER = "\\|"
```

Comment:
Jim usually links to [this](https://ccristudio.jira.com/wiki/pages/viewpage.action?spaceKey=STEALTH&title=Keywords) as the compelling use case not to. 

It does restrict us in the command line. We also discussed maybe only allowing adding one keyword at a time